modules/core/arrow-core-data/src/test/kotlin/arrow/core/ConstTest.kt

```diff
@@ -11,16 +11,22 @@ import arrow.test.laws.EqLaws
 import arrow.test.laws.ShowLaws
 import arrow.test.laws.TraverseFilterLaws
 import arrow.typeclasses.Eq
+import io.kotlintest.properties.Gen
 
 class ConstTest : UnitSpec() {
   init {
     Int.monoid().run {
       testLaws(
         TraverseFilterLaws.laws(Const.traverseFilter(), Const.applicative(this), { Const(it) }, Eq.any()),
         ApplicativeLaws.laws(Const.applicative(this), Eq.any()),
-        EqLaws.laws(Const.eq<Int, Int>(Eq.any())) { Const(it) },
-        ShowLaws.laws(Const.show(), Const.eq<Int, Int>(Eq.any())) { Const(it) }
+        EqLaws.laws(Const.eq<Int, Int>(Eq.any()), Gen.const(Gen.int())),
+        ShowLaws.laws(Const.show(), Const.eq<Int, Int>(Eq.any()), Gen.const(Gen.int()))
       )
     }
   }
 }
+
+private fun <A> Gen.Companion.const(gen: Gen<A>): Gen<Const<A, A>> =
+  gen.map {
+    Const<A, A>(it)
+  }
```

Comment:
I think this would fit better if it was parameterized to `A` and `B` since they don't have to be the same

Reply:
Also let's move this and the other common generators to the generators file

Reply:
ok

which other generators do you mean?  in the other cases it does not seem to make sense to move it

Reply:
Wrote this thinking there was another one, but you are right, that is the only one that makes sense :+1: